### PR TITLE
WIP logging: add stdout handler

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: qatrfm/utils/logger.py [hunk 1]
```diff
@@ -62,7 +63,11 @@ def warning(self, msg, *args, **kwargs):
     @staticmethod
     def getQatrfmLogger(name):
         logging.setLoggerClass(QaTrfmLogger)
-        return logging.getLogger(name)
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setLevel(logging.DEBUG)
+        logger = logging.getLogger(name)
+        logger.addHandler(handler)
+        return logger
```
Comment (jlausuch) [inline qatrfm/utils/logger.py:71]: To be more complete, we can add:

> formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
> handler = logging.StreamHandler()
> handler.setLevel(level=logging.DEBUG)
> handler.setFormatter(formatter)

File: qatrfm/utils/logger.py [hunk 2]
```diff
@@ -62,7 +63,11 @@ def warning(self, msg, *args, **kwargs):
     @staticmethod
     def getQatrfmLogger(name):
         logging.setLoggerClass(QaTrfmLogger)
-        return logging.getLogger(name)
+        handler = logging.StreamHandler(sys.stdout)
```
Comment (jlausuch) [inline qatrfm/utils/logger.py:66]: not sure you need to specify here `sys.stdout`

**Comment by asmorodskyi** [comment]
low prio for now , so we investigate this later

**Comment by asmorodskyi** [comment]
I combine this commit with another one which also need and create another PR , so closing this one

**Pull request closed without merging**